Galleries/art.html

```diff
@@ -489,7 +490,7 @@ <h3>Contribute</h3>
               <hr>
               <a href="#Top">Back to Top</a>
               <br>
-              <a href="https://www.DesignXDestiny.com/"><img src="./pinklogo.png" alt="Home"></a>
+              <a href="https://www.DesignXDestiny.com/"><img src="./images/pinklogo.png" alt="Home"></a>
```

Comment:
# Fix This!
This logo needs another dot to find its way up a level to the images folder.